guides/installation_guides/c++/mac/README.md

```diff
@@ -1,9 +1,34 @@
-# Cosmos Guides
+# Cosmos Guides: Getting ready to program in C++ in MAC OS
 > Your personal library of every algorithm and data structure code that you will ever encounter
+ 
+  **Getting The Software** </br>
+  There are many IDE's for MAC OS but here's a quick guide to install one of them i.e **CODEBLOCKS**.</br>
+  You can get Codeblocks from the Official Site: http://www.codeblocks.org/downloads. Click on **Download the binary release** <br><br>
+  <img src="https://github.com/Nidakhan786/cosmos/blob/installation-guide/guides/images/1%20(1).png"> <br><br>
```

Comment:
You must link the images you have committed in this pull request that is, replace `Nidakhan786` by `OpenGenus`.